specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -5318,543 +6439,463 @@
-            "OnlyFreeAllowed",
-            "NotAllowed"
-          ],
-          "x-ms-enum": {
-            "name": "MarketplacePurchasesPolicy",
-            "modelAsString": true
-          }
-        },
-        "reservationPurchases": {
-          "description": "The policy that controls whether Azure reservation purchases are allowed for a billing profile.",
+        "recipientEmailId": {
           "type": "string",
-          "enum": [
-            "Allowed",
-            "NotAllowed"
-          ],
-          "x-ms-enum": {
-            "name": "ReservationPurchasesPolicy",
-            "modelAsString": true
-          }
+          "description": "The email ID of the recipient to whom the transfer request is sent."
         },
-        "viewCharges": {
-          "description": "The policy that controls whether users with Azure RBAC access to a subscription can view its charges.",
+        "resellerId": {
           "type": "string",
-          "enum": [
-            "Allowed",
-            "NotAllowed"
-          ],
-          "x-ms-enum": {
-            "name": "ViewChargesPolicy",
-            "modelAsString": true
-          }
+          "description": "Optional MPN ID of the reseller for transfer requests that are sent from a Microsoft Partner Agreement billing account."
         }
       }
     },
-    "CustomerPolicy": {
-      "description": "The customer's Policy.",
+    "AcceptTransferRequest": {
       "type": "object",
-      "allOf": [
-        {
-          "$ref": "#/definitions/Resource"
-        }
-      ],
+      "description": "Request parameters to accept transfer.",
       "properties": {
         "properties": {
+          "description": "Request parameters to accept transfer.",
           "x-ms-client-flatten": true,
-          "$ref": "#/definitions/CustomerPolicyProperties",
-          "title": "Customer Policy properties"
+          "$ref": "#/definitions/AcceptTransferProperties"
         }
       }
     },
-    "CustomerPolicyProperties": {
-      "description": "The properties of a customer's policy.",
+    "AcceptTransferProperties": {
+      "type": "object",
+      "description": "Request parameters to accept transfer.",
       "properties": {
-        "viewCharges": {
-          "description": "The policy that controls whether the users in customer's organization can view charges at pay-as-you-go prices.",
-          "type": "string",
-          "enum": [
-            "Allowed",
-            "NotAllowed"
-          ],
-          "x-ms-enum": {
-            "name": "ViewCharges",
-            "modelAsString": true
+        "productDetails": {
+          "description": "Request parameters to accept transfer.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ProductDetails"
           }
         }
       }
     },
-    "AvailableBalance": {
-      "description": "The latest Azure credit balance. This is the balance available for pay now.",
+    "TransferDetails": {
```

Comment:
transferDetails is missing the top level "id", "name", and "type" properties

Reply:
I have added Arm properties now.